app/views/submissions/_version_links.html.erb

```diff
@@ -3,13 +3,15 @@
     <%= link_to(url_for([:view, @course, @assessment, @prevVersion[:submission], header_position: @prevVersion[:header_position]]), class: "btn small", title: "Previous version containing this file (shortcut: [)", id: "prev_version_link") do %>
       <i class="material-icons">south</i>
     <% end %>
+    <link rel="prerender" href=<%= url_for([:view, @course, @assessment, @prevVersion[:submission], header_position: @prevVersion[:header_position]]) %> />
```

Comment:
I don't think we should be prerendering the next/prev version pages - it's a resource intensive suggestion & we are already prerendering the left & right students.

Reply:
Seems like it won't be resource intensive.
https://www.chromium.org/developers/design-documents/prerender/#minimizing-resource-contention
but it means only 2 out of our 4 pages will be prerendered